types/entities.ts

```diff
@@ -104,6 +110,8 @@ export interface WorkSummary {
     published: number;
     total: number | null;
   };
+  series: BasicSeries[];
+  seriesIds: BasicSeries["id"][];
```

Comment:
Since this can be obtained just by `array.map`'ing over the series array (unless I'm missing something), I wouldn't add both

Reply:
Sounds good!